share/completions/pactl.fish

```diff
@@ -67,7 +67,7 @@ complete -f -c pactl -n "not __fish_seen_subcommand_from $commands" -a remove-sa
 complete -f -c pactl -n "__fish_seen_subcommand_from play-sample remove-sample" -a '(__fish_pa_complete_type samples)'
 
 complete -f -c pactl -n "__fish_seen_subcommand_from unload-module" -a '(__fish_pa_complete_type modules)'
-complete -f -c pactl -n "__fish_seen_subcommand_from load-module" -a '(__fish_pa_complete_unloaded_modules)'
+complete -f -c pactl -n "__fish_seen_subcommand_from load-module; and command -sq pulseaudio" -a '(__fish_pa_complete_unloaded_modules)'
```

Comment:
This is the wrong place for that condition, it should be checked *inside* `__fish_pa_complete_unloaded_modules`.

At least conceptually, that could be called from multiple places. Also, the complet condition here should still apply even if no pulseaudio exists to let the `--no-files` take effect.

Reply:
Yeah sorry I guess I took the easy way out